packages/main/src/plugin/extension-loader.ts

```diff
@@ -57,6 +57,7 @@ import { securityRestrictionCurrentHandler } from '../security-restrictions-hand
 import type { IconRegistry } from './icon-registry.js';
 import type { Directories } from './directories.js';
 import { isLinux, isMac, isWindows } from '../util.js';
+import { runCliCommand } from './util/exec.js';
```

Comment:
if file is named exec probably we should have command named `exec`

Reply:
updated, now the method name calls `exec` instead of `runCliCommand `